translate/rus.json

```diff
@@ -295,7 +296,7 @@
 "Please consider whitelisting [TCEC_URL] in your ad blocker --- or send a donation.":"Пожалуйста, добавьте в белый список [TCEC_URL] в вашем блокировщике рекламы --- или отправьте пожертвование.",
 "points + direct encounters":"очки + счёт личных встреч",
 "points":"очки",
-"popup right click":"всплывающее правой кнопкой",
+"popup right click":"контекстное меню правой кнопкой",
```

Comment:
For this one, I am not sure whatever it will fit in terms of size in 2 rows. If you want, we can discuss it. And the word 'контекстное' works here just as good as the previous translation.